src/components/Card/Card.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300&display=swap');
+
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  color: #000000;
+}
+
+.wrapper {
+  margin: 2rem;
+  display: grid;
+  gap: 1rem;
+  grid-template-columns: 1fr;
+  justify-content: center;
+}
+
+.Card {
+  border-radius: 0.2rem;
+  height: 16rem;
+  display: flex;
+  position: relative;
+  overflow:hidden;
+  background-color:#f8f8f8;
```

Comment:
Could make things more manageable and readable if you use CSS vars for colours?

Reply:
Yes I will be using varables on in the final doc